rust-connectors/sinks/kafka/src/main.rs

```diff
@@ -1,13 +1,19 @@
 use fluvio_connectors_common::git_hash_version;
-use fluvio_connectors_common::opt::{CommonConnectorOpt, Record};
-use fluvio_future::tracing::{debug, info};
+use fluvio_connectors_common::opt::CommonConnectorOpt;
+use fluvio_future::tracing::{error, info};
+use rdkafka::config::ClientConfig;
+use rdkafka::producer::{FutureProducer, FutureRecord};
 use schemars::schema_for;
 use schemars::JsonSchema;
 use structopt::StructOpt;
 use tokio_stream::StreamExt;
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
+    // =======================================================
```

Comment:
This comment block seems redundant to me. 
If we want to comment on something, let's just do it without graphical wrappings.

Reply:
removing